packages/drivers/routerlicious-driver/package.json

```diff
@@ -109,6 +109,10 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
+			"InterfaceDeclaration_IRouterliciousDriverPolicies": {
+				"backCompat": false
+			}
+		}
```

Comment:
Changes that aren't backwards compatible should be made in next.

Reply:
Will open a PR in next. Since this was not used outside or was created inside FF and was also optional, so I thought it is ok to do it in main. But I think we should not do this just to keep sanity.

Reply:
Opened here; https://github.com/microsoft/FluidFramework/pull/16749